trade-finance-manager-ui/templates/utilisation-reports/partials/premium-payments-html.njk

```diff
@@ -0,0 +1,132 @@
+{% if premiumPayments.showMatchSuccessNotification %}
+  {% set notificationHtml %}
+    <h3 class="govuk-notification-banner__heading" data-cy="match-success-notification-heading">
+      Match payment recorded
+    </h3>
+    <p class="govuk-body" data-cy="match-success-notification-message">The fee(s) are now at a Match state. Further payments cannot be added to the fee record.</p>
+  {% endset %}
+
+  <div class="govuk-grid-row govuk-!-padding-top-7" data-cy="match-success-notification">
+    <div class="govuk-grid-column-one-half-from-desktop">
+      {{ govukNotificationBanner({
+        html: notificationHtml,
+        type: "success"
+      }) }}
+    </div>
+  </div>
+{% endif %}
+
+{% if premiumPayments.tableDataError or premiumPayments.filterError %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: [premiumPayments.tableDataError, premiumPayments.filterError],
+    attributes: {
+      'data-cy': 'error-summary'
+    }
+  }) }}
+{% endif %}
+
+{% set clearButtonHref %}
+    /utilisation-reports/{{ reportId }}
+{% endset %}
+  <h2 class="govuk-heading-l" data-cy="premium-payments-heading">Premium payments</h2>
+
+  <div class="govuk-grid-row premium-payments-description-filter-container">
+    <div class="govuk-grid-column-one-half premium-payments-description-container">
+      {% if userCanEdit %}
+        <p class="govuk-body" data-cy="how-to-add-payments-text">Received payments are entered against reported fees through selection and then selection of the 'Add a payment' button.</p>
+
+        <p class="govuk-body" data-cy="how-to-generate-keying-data-text">
+          When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when the 'Generate keying data' button is selected.
+        </p>
+      {% else %}
+        <p class="govuk-body" data-cy="received-payments-text">Received payments are entered against reported fees. When payments show as matched, the adjustment data for keying into ACBS will be automatically generated.</p>
+      {% endif %}
+    </div>
+
+    <div class="govuk-grid-column-one-half govuk-!-text-align-right premium-payments-filter-container">
+      <form method="GET" data-cy="facility-filter-form">
```

Comment:
What is the `data-cy` for?

Reply:
not sure, it's just been moved (was not made by me)

Reply:
Ugh, it's used in 2x places to assert the form **contains** some text 😞